### ISSUE TITLE
[Low Priority] Warning on sensor on Kado popup on Desktop ( Not Mobile )

### ISSUE THREAD

**Issue by apollo-pablo** [wynddaoOld/dex-issue-tracker#44]
> Problem

On Desktop, the following warn appear ( blocked by security ). Its not a problem, but something indicates that it tries to use code design for Mobile.

![image](https://user-images.githubusercontent.com/95628259/212551644-e9accba8-b519-498a-9001-519cb1f282fe.png)
